styles/styles.css

```diff
@@ -0,0 +1,213 @@
+* {
+  box-sizing: border-box;
+}
+button, input, select {
+  font-family: inherit;
+  font-size: 100%;
+}
+header {
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+}
+footer {
+  color: white;
+  background-color: rgb(13, 45, 78);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3rem 0 3rem;
+}
+body {
+  margin: auto;
+  font-size: 16px;
+}
+.logo {
+  padding: 3rem 0rem 0rem 6rem;
+  width: 100%;
+}
+.tagline {
+  font-size: 1.55rem;
+  margin-top: -1rem;
+  padding: 0rem 0rem 10rem 6rem;
+  color: rgb(116, 171, 42);
+}
+.cost {
+  color: white;
+}
+.checkout {
+  color: white;
+  font-weight: bold;
+  background-color: rgb(116, 171, 42);
+  padding: .01rem 2rem;
+  border-radius: .1rem;
+}
+.checkout-link {
+  text-decoration: none;
+}
+.cart {
+  width: 15%;
+  padding-right: .5rem;
+}
+.cart-cost {
+  display: flex;
+  justify-content: flex-end;
+}
+.cart-cost-button {
+  padding: 3rem 6rem 0rem 0rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+.top-nav {
+  background-color: rgb(231, 231, 231);
+  width: 100%;
+  padding: .5rem 0rem .5rem 12rem;
+}
+.top-nav a {
+  text-decoration: none;
+  color: rgb(11, 38, 66);
+}
+li {
+  display: inline;
+  text-decoration: none;
+  font-weight: bold;
+}
+li::after {
+  content: "/";
+  padding: .8rem;
+}
+li:last-of-type:after {
+  content: none;
+}
+ul {
+  padding-left: 0;
+}
+.header-flexbox {
+  display: flex;
+  justify-content: space-between;
+}
+.logo-tagline {
+  width: 40%;
+}
+.footer-logo {
+  width: 25%;
+}
+h2 {
+  padding: 1rem 0rem;
+}
+.h2-padding {
+  padding: 1rem 0rem 1rem 12rem;
+}
+.dogs-main {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 2rem 12rem 3rem;
+}
+.first-main-image {
+  padding-right: 2.5rem;
+  width: 40%;
+}
+.home-dogs {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 0rem 9.625rem 2rem;
+}
+.dog-card {
+  width: 20%;
```

Comment:
the width of 20% is a little hard at smaller widths. Since they will wrap to the next row if they overflow the wrapper, I'd recommend setting a min-width so they don't get so skinny when your browser is narrower. As well, it might be a good idea to add a max width to the app itself so it doesn't look weird on wide monitors.